src/ripe_rainbow/interactive/drivers.py

```diff
@@ -64,13 +67,21 @@ def find_elements_by_css_selector(self, selector):
     def find_element_by_name(self, name):
         return self.find_by_name(name)
 
-    def click_element(self, element):
-        try: from selenium.webdriver.common.action_chains import ActionChains
-        except ImportError: ActionChains = None
-
-        ActionChains(self.instance).move_to_element(element).click(element).perform()
-
-        return element
+    def focus(self, element):
+        from selenium.webdriver.common.action_chains import ActionChains
+        actions = ActionChains(self.instance)
+        actions.move_to_element(element)
+        actions.perform()
+
+    def click(self, element, focus = True):
+        from selenium.webdriver.common.action_chains import ActionChains
+        if focus:

```

Comment:
Maybe we could reuse focus from above, shouldn't make any difference functionality-wise